index.html

```diff
@@ -126,6 +126,27 @@ <h4 class="card-title">Rules</h4>
                                     <li>If a car is next to you (at least half a car alongside) before a braking zone, they have the right to be there when the corner comes up. They should leave you room and you should leave them room (which will normally mean you both will go slower than usual). Either of you are allowed to safely back out but are not obliged to. Please see <a href="https://f1metrics.wordpress.com/2014/08/28/the-rules-of-racing/">https://f1metrics.wordpress.com/2014/08/28/the-rules-of-racing/</a> for more information</a></li>
                                 </ol>
                             </li>
+                            <li>
+                                <strong>REJOINING THE TRACK</strong>
+                                <ol>
+                                    <li>
+                                        If you leave the track (all 4 wheels outside of the white line) you must rejoin the track safely, this will mean:
+                                        <ol>
+                                            <li>You must be in control of the car, rather than just pointing the car to the road and hoping for the best</li>
+                                            <li>Once joined the track, you must stay predictable if there are cars near you. Especially when your going slower. You must not impede other drivers while you are recovering to the track</li>
+                                            <li>You must only rejoin the track if it is safe to do so, you must not cause another car to take avoiding action to prevent contact</li>
+                                            <li>You are deemed to have recovered to the track once you have regained full control of the vehicle, reached a safe speed for that part of the track and are on a predictable line</li>
+                                            <li>You should wait until a reasonable opportunity arises to enter the track. If an approaching car has a reasonable ability to give you room (IE they are not approaching or exiting a corner) you can rejoin the track with caution. If they are fighting for position against another car you should let them pass</li>
+                                        </ol>
+                                    </li>
+                                    <li>
+                                        If you see another car off-track:
+                                        <ol>
+                                            <li>If a car is off-track you should do reasonable steps to ensure there is track space for them to rejoin the track. e.g: If they went off track to the left, then go to the right side of the road if possible</li>
```

Comment:
Probably change 'do' to 'take' `you should do reasonable steps` to `you should take reasonable steps`